### ISSUE TITLE
Bean autowire issue while using FallbackFactory

### ISSUE THREAD

**Issue by paro87** [Playtika/feign-reactive#170]
I'm getting an error (Could not autowire. No beans of ServiceFeignClient type found.) while including FallbackFactory to the parameters as:

```java
@ReactiveFeignClient(name = "service_statistics", fallbackFactory = StatisticFallbackFactory.class)
public interface ServiceFeignClient {
    ...
}
```
I realized that in OpenFeign FallbackFactory comes from feign.hystrix.FallbackFactory package, whereas in Reactive one, it comes from reactivefeign.FallbackFactory. Maybe there is some Hystrix support problem or bean conflict that causes that error. But it it Ok when using only Fallback class.

**Comment by kptfh** [comment]
It's expected that you have the instance of StatisticFallbackFactory bean in the context.
[Here is the test for this case](https://github.com/Playtika/feign-reactive/blob/develop/feign-reactor-test/feign-reactor-spring-configuration-test/src/test/java/reactivefeign/spring/config/AutoConfigurationTest.java#L175)
If it won't fix your issue then add stacktrace here. 

Also we may consider to instantiate FalbackFactory if it's not present in the context. Can you suggest how it works in OpenFeign?

**Comment by paro87** [comment]
I used it in my project according to this example (3rd example).
[https://medium.com/@kirill.sereda/spring-cloud-netflix-hystrix-%D0%BF%D0%BE-%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%B8-e60e91a6770f](url)

**Comment by kptfh** [comment]
fixed in 2.0.0
